Решение домашнего задания.

Arrowin24 opened the pull request:
Здесь представлены все 3 домашних задания по уроку Наследования.

Arrowin24 committed:
Решение домашнего задания.

DmitryN10 commented on src/transport/Transport.java:
```diff
@@ -0,0 +1,72 @@
+        this("default", "default", 2000, "default", "Black", 50);
+    }
+
+    public Transport(String brand, String model, int yearIssue, String country, String color, double maxSpeed) {
```
Игорь, не совсем так. Не нужен пустой конструктор. Ты должен проверять входные парметры в конструкторе. И в случае null или пусто строки, заполнять значение по умолчанию - "default"

Аналогично и в других классах.

В остальном вопросов нет по это задаче

DmitryN10 commented on src/animals/Animal.java:
```diff
@@ -0,0 +1,36 @@
+    }
+
+    public abstract void eat();
+    public abstract void sleep();
```
Здесь еще должен быть метод перемещаться

Arrowin24 committed:
1. Исправил конструктор класса Transport(). Теперь он проверяет строку на допустимое значение.
2. Добавил абстрактный метод Перемещаться (move()). Реализовал его в дочерних классах.
